pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -653,18 +653,34 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 					PropertyName: "httpMethod",
 				},
 				{
-					Label:        "Username",
+					Label:        "HTTP Basic Authentication - Username",
 					Element:      alerting.ElementTypeInput,
 					InputType:    alerting.InputTypeText,
 					PropertyName: "username",
 				},
 				{
-					Label:        "Password",
+					Label:        "HTTP Basic Authentication - Password",
 					Element:      alerting.ElementTypeInput,
 					InputType:    alerting.InputTypePassword,
 					PropertyName: "password",
 					Secure:       true,
 				},
+				{ // New in 9.1
+					Label:        "Authorization Header - Scheme",
+					Description:  "Optionally provide a scheme for the Authorization Request Header. Default is Bearer.",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					PropertyName: "authorization_scheme",
+					Placeholder:  "Bearer",
+				},
+				{ // New in 9.1
+					Label:        "Authorization Header - Credentials",
+					Description:  "Credentials for the Authorization Request header. Only one of HTTP Basic Authentication or Authorization Request Header can be set.",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
```

Comment:
Should we use `InputTypePassword` here?

Reply:
I debated on this before opening the PR, and in the end, I settled on no.

First, I took a look at Slack as a precedent as I remember webhook URLs or Tokens as examples are meant to be secret too, but they're not obscured. They are stored encrypted though, which we've done here too.

https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/notifier/available_channels.go#L401-L408

Second, unlike a password, tokens are very long strings which are hard at a glance to remember (even though a password can be as well, but it's more about "what's common"), and in the scenario where you have multiple tokens for a webhook in can be helpful to know which is which.

Let me know if you have other thoughts or if you think otherwise!